install

```diff
@@ -21,18 +21,22 @@ main() {
   #local exec_filename=oil.ovm-dbg
   local exec_filename=oil.ovm  # Should this go in configure output?
 
-  local dest="$PREFIX/bin/$exec_filename"
+  local BINDIR="${PREFIX}/bin"
 
-  echo "Installing executables in $dest"
-  if ! install -v _bin/$exec_filename $dest; then
+  echo "Installing executables in ${DESTDIR}/${BINDIR}"
```

Comment:
You don't need `${DESTDIR}` as opposed to `$DESTDIR` in any of these places.  Please remove it and revert all the changes to that add `${}`.

There are some unnecessary double quotes too but I guess those are OK for consistency, and do change the meaning.  The ${} doesn't change anything.

Reply:
There are very few unnecessary double-quotes in shell scripts and especially under the aspect of robustness I'd stick to quoting everything.